locales-src/newtab.ftl

```diff
@@ -19,3 +19,20 @@ newtab-search-box-search-the-web-text = Search the Web
 newtab-search-box-search-the-web-input =
     .placeholder = Search the Web
     .title = Search the Web
+
+## Top Sites form dialog
+
+newtab_topsites_add_header = New Top Site
+newtab_topsites_edit_header = Edit Top Site
+newtab_topsites_title_label = Title
+newtab_topsites_title_placeholder =
```

Comment:
nit: fluent convention seems to be `-` dashes instead of `_` underscores

Reply:
Indeed. Also, [this doc](https://docs.google.com/document/d/12QdVuKgJpZ39YZYt5sOZWqAGOXYyw_gfl4hbgY9AX0A/edit#) might be a useful pointer (I'm starting to share it around before landing it in-tree)